PCbuild/_tkinter.vcxproj

```diff
@@ -112,6 +113,7 @@
     <_TclTkDLL Include="$(tcltkdir)\bin\$(tclDllName)" />
     <_TclTkDLL Include="$(tcltkdir)\bin\$(tkDllName)" />
     <_TclTkDLL Include="$(tcltkdir)\bin\$(tclZlibDllName)" />
+    <_TclTkDLL Include="$(tcltkdir)\bin\libtommath.dll" Condition="$(TclMajorVersion) == '9'"/>
```

Comment:
Do we have to redistribute this file with Tcl/Tk 9.0? If so, there'll be updates under `Tools/msi` as well (look for references to tclZlibDllName).

Also, we should be using the name property here rather than the filename directly.

Reply:
Once we're using the name property, checking whether the name is empty rather than checking the version is going to be more robust.

```suggestion
    <_TclTkDLL Include="$(tcltkdir)\bin\$(tommathDLLName)" Condition="$(tommathDLLName) != ''"/>
```

Reply:
Now using names, not paths and checking whether the name is empty in the Condition.